api/apps/api/src/modules/projects/projects-crud.ts

```diff
@@ -34,7 +32,7 @@ type ProjectFilterKeys = keyof Pick<
 type ProjectFilters = Record<ProjectFilterKeys, string[]>;
 
 @Injectable()
-export class ProjectsService extends AppBaseService<
+export class ProjectsCrud extends AppBaseService<
```

Comment:
I suggest to stick to a qualified noun for these classes (I wouldn't consider CRUD to be a noun but more of an adjective, if anything) - `ProjectsCrudService`? `ProjectsBaseService`?

Reply:
Almost everything can be/is a service thus I feel it may be redundant but sure, we may stick to this convention 👍

Reply:
https://github.com/Vizzuality/marxan-cloud/pull/218/commits/f48dc4e0f057e25b2366b059a4fbcf55ee6e103a